Tests/RequestTests.cs

```diff
@@ -9,53 +9,231 @@ namespace Tests
     [TestFixture]
     public class RequestTests
     {
-        private ZendeskApi api = new ZendeskApi(Settings.Site, Settings.Email, Settings.Password);
+        private readonly ZendeskApi _api = new ZendeskApi(Settings.Site, Settings.Email, Settings.Password);
 
         [Test]
-        public void CanGetRequests()
+        public void CanGetAllRequests()
         {
-            var res = api.Requests.GetAllRequests();
+            var res = _api.Requests.GetAllRequests();
             Assert.True(res.Count > 0);
-        }        
+        }
+
+        [TestCase(1, 1)]
+        [TestCase(1, 2)]
+        public void CanGetAllRequestsPaged(int perPage, int page)
+        {
+            Assert.DoesNotThrow(() =>
```

Comment:
`GetAllRequests` can throw an exception. It should not typically throw an exception, but if it does, it should throw an assertion.